src/pages/index.js

```diff
@@ -127,19 +259,38 @@ profileEditButton.addEventListener("click", () => {
 
 /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+avatarEditButton.addEventListener("click", () => handleAvatarModalOpen());
+avatarModalCloseButton.addEventListener("click", () =>
+  handleAvatarModalClose()
+);
+avatarModalSubmitButton.addEventListener("submit", (event) => {
+  console.log(avatarModalSubmitButton);
+  return handleAvatarUpdate();
+});
+
+deleteModalButton.addEventListener("click", () => {
```

Comment:
Any popup should be created using a class. And click handlers should be added inside the class